### PR TITLE
fix typos in docs

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Sun Hongliang <allen.sun@daocloud.io>

### CODE REVIEW

File: docs/plan-for-production.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Swarm clusters in business critical production environments. The following high
 
 ## Security
 
-There are many aspects to securing a Docker Swarm cluster. This section covers:
+There are many aspects to secure a Docker Swarm cluster. This section covers:
```
Comment (nishanttotla) [inline docs/plan-for-production.md]: I think `securing` is actually correct here.

**Comment by nishanttotla** [comment]
Ping @moxiegirl

File: docs/plan-for-production.md (hunk 1 above)
Comment (moxiegirl) [inline docs/plan-for-production.md]: `securing` is correct

File: docs/plan-for-production.md [hunk 2]
```diff
@@ -60,16 +60,16 @@ the different components of a Swam cluster listen on. You should use these to
 configure your firewalls and other network access control lists.
 
 - **Swarm manager.**
-    - **Inbound 80/tcp (HTTP)**. This is allows `docker pull` commands to work. If you will be pulling from Docker Hub you will need to allow connections on port 80 from the internet.
+    - **Inbound 80/tcp (HTTP)**. This allows `docker pull` commands to work. If you will be pulling from Docker Hub you will need to allow connections on port 80 from the internet.
```
Comment (moxiegirl) [inline docs/plan-for-production.md]: Change 

```
If you will be pulling from Docker Hub you will need to allow connections on port 80 from the internet
``

to

```
If you plan to pull from Docker Hub, you must allow Internet connections to port 80.
```

**Comment by moxiegirl** [comment]
Thanks for the contribution @allencloud --- some small corrections.

**Comment by allencloud** [comment]
Ping @moxiegirl

**Comment by thaJeztah** [comment]
LGTM :+1:

**Comment by moxiegirl** [comment]
Thanks @allencloud !

**Pull request merged**